compiler/luci/lang/include/luci/IR/Nodes/CircleConst.h

```diff
@@ -34,6 +35,9 @@ namespace luci
 class CircleConst final : public FixedArityNode<0, CircleNodeImpl<CircleOpcode::CIRCLECONST>>
 {
 public:
+  CompressionType compression(void) const;
+  void compression(CompressionType c);
```

Comment:
move to below of `template <loco::DataType DT> typename loco::DataTypeImpl<DT>::Type &scalar(void);`